CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
```

Comment:
Could you remove the `Unreleased` line, as otherwise the [auto-release workflow](https://github.com/alphagov/govuk-infrastructure/blob/a6d64e5e8c0514199d6b00ee3938af224fb1b527/.github/workflows/autorelease-rubygem.yml#L77) will put the next version above this.

Reply:
Cool, done! 👍